src/app/docs/selection-box/selection-box-docs.component.html

```diff
@@ -0,0 +1,85 @@
+<sky-docs-demo-page
+  moduleName="SkySelectionBoxModule"
+  moduleSourceCodePath="src/app/public/modules/selection-box/"
+  pageTitle="Selection box"
+>
+  <sky-docs-demo-page-summary>
+    <!-- Page summary goes here -->
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+    >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-radio-group
+          heading="Form type"
```

Comment:
This label seems off to me. @Blackbaud-AdamFunderburk might have insight if my suggestion seems off.
```suggestion
          heading="Selection type"
```